ticker/src/test/java/com/robinhood/ticker/TickerColumnManagerTest.java

```diff
@@ -83,4 +83,12 @@ public void test_shouldDebounce() {
         assertTrue(tickerColumnManager.shouldDebounceText("1234".toCharArray()));
         assertFalse(tickerColumnManager.shouldDebounceText("12345".toCharArray()));
     }
+
+    private TickerColumn tickerColumnAtIndex(int index) {
```

Comment:
Was not sure of what naming conventions y'all use so I just tried to make it read off like english. Open to suggestions